features/no_title_info_message.feature

```diff
@@ -0,0 +1,19 @@
+@US316
+Feature: Explain why some properties do not have title information
+
+
+As a customer
+I want to know why some properties do not have title information
+So that I can understand why the information that I want is not available through the DRV service
+
+Add a line to why not all properties are registered (to the 'No title information found -
+Why not?' help text???)
+
+Acceptance criteria:
+Confirm that properties in the results list where no title information is found has a link to an explanation of why this might have occurred
+Confirm that the link in the explanation links to the FaP search page
```

Comment:
@juliadart The feature and step definitions for this acceptance criteria do not include a test for the FaP link, but they should.